extension/src/background/index.ts

```diff
@@ -24,25 +26,30 @@ export const initContentScriptMessageListener = () => {
         file: "contentScript.min.js",
       });
     }
+    return undefined;
```

Comment:
looks like we should be returning `true | undefined` here, returning true means that there is an async response expected so I believe in our case `undefined` is the correct return value.

Reply:
We do `await` a response from the content script, so I think maybe `true` would be correct here?

https://github.com/stellar/freighter/blob/master/%40shared/api/external.ts#L15

Reply:
I _think_ they mean that in the case where you got an event in your listener, and that listener's callback plans to `sendResponse` async then you should return `true` to signal that.
https://stackoverflow.com/questions/70353944/chrome-runtime-onmessage-returns-undefined-even-when-value-is-known-for-asynch